tests/integration/cqlengine/query/test_limit.py

```diff
@@ -0,0 +1,96 @@
+# Copyright 2013-2017 DataStax, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+try:
+    import unittest2 as unittest
+except ImportError:
+    import unittest
+
+from cassandra.cluster import Cluster
+from cassandra.cqlengine.models import Model
+from cassandra.cqlengine import columns
+from cassandra import util
+from cassandra.cqlengine import connection
+import cassandra
+import datetime
+
+from cassandra.cqlengine.management import create_keyspace_simple
+from tests.integration import PROTOCOL_VERSION
+
+## Create DB object
+class test_table(Model):
+
+    __keyspace__ = "test_keyspace"
+    user = columns.Integer(primary_key=True)
+    prop = columns.Text()
+
+# Test
+class LimitTest(unittest.TestCase):
+
+    def setUp(self):
```

Comment:
If this tests was included in the file and in the appropriate class like TestQuerySetOperation it would change quite a bit how it's done, you can see how other tests are done there but there wouldn't be need for all this